glommio/src/executor/mod.rs

```diff
@@ -818,6 +799,16 @@ impl PoolThreadHandles {
         self.handles.push(handle)
     }
 
+    /// Returns the number of threads that were successfully spawned.
+    pub fn nr_spawn_ok(&self) -> usize {
+        self.handles.iter().filter(|r| r.is_ok()).count()
+    }
+
+    /// Returns the number of threads that failed to spawn.
+    pub fn nr_spawn_err(&self) -> usize {
```

Comment:
I am not sure I am too hot on this, but I see the point.
Ideally, if one of the threads failed to spawn, we should fail the whole thing.

It doesn't seem that easy to do, as we can't just panic, but maybe we can `cancel()` the successful handles.

The challenge is separate failed execution - which is fine - from failed spawn.

Reply:
I'll revise this as well -- I plan on having all threads wait until they have all been successfully spawned; if any fail to spawn, those that have spawned will exit without ever creating their local executor.  Hopefully, this way the user won't have to wonder about what has and has not been executed so far.  Let me know if you prefer a different approach.